src/views/portals/YetiView.vue

```diff
@@ -1,4 +1,5 @@
 <template>
+<!-- eslint-disable -->
```

Comment:
This: https://github.com/vuejs/eslint-plugin-vue/issues/926

Reply:
OK, then this should be removed once the i18n is in place, because there should be not such problem anymore.
Anyway, if you need to disable a rule somewhere, do disable it only where it applies (preferably at the line level), not for the whole file.

Reply:
@iamvdo, can you desactivate it at line level please? I understand the use-case, but disabling entire linting at file level will increase the maintenance cost.  

You can simply add this comment above line that causes issue : 

```html
<!-- eslint-disable-next-line no-irregular-whitespace -->
```

Reply:
Other possibility, if there is too many issue with `no-irregular-whitespace`, it make senses to disable this specific rule at file level, waiting for i18n PR. But please keep other rules enabled.

Reply:
Linting is not disabled at file level, but only inside `<template>` tag.
Btw, with i18n, this will be out-of-date soon.

For this PR, two solutions:

- merge as-is
- remove proper typographic characters before merge

Let me know

Reply:
Does this at template level fix the issue ? 

```
<!-- eslint-disable no-irregular-whitespace -->
```